research/conditional/moe_layers/continuous_moe.py

```diff
@@ -172,47 +173,48 @@ def log_heavy(self):
         log = {}
         if self.group_size == 1:
             return log
-        merge_weights = torch.flatten(
-            self.logging_cache["merge_weights"], start_dim=0, end_dim=-2
-        )
-        merge_logits = torch.flatten(
-            self.logging_cache["merge_logits"], start_dim=0, end_dim=-2
-        )
-        sample_weight_distros = merge_weights[:5]
-        sample_logits_distros = merge_logits[:5]
-
-        for i, sample in enumerate(sample_weight_distros):
-            sample = sample.sort(descending=True).values
-            sample = sample.tolist()
-            fig = px.bar(x=range(len(sample)), y=sample, title=f"sample {i}")
-            log[f"merge_weights/sample_{i}"] = fig
-
-        for i, sample in enumerate(sample_logits_distros):
-            sample = sample.sort(descending=True).values
-            sample = sample.tolist()
-            fig = px.bar(x=range(len(sample)), y=sample, title=f"sample {i}")
-            log[f"merge_logits/sample_{i}"] = fig
```

Comment:
we losing this image, is it intentional?

Reply:
yes we dropping this boi